confluence.sh

```diff
@@ -25,7 +26,7 @@ Examples:
 # check java 8
 javaVersion=`java -version 2>&1 | head -n 1 | cut -d\" -f 2`
 javaCompilerVersion=`javac -version 2>&1 | head -n 1 | cut -d\" -f 2`
-[[ "$javaVersion" != "1.8."* || "$javaCompilerVersion" != *"1.8."* ]] && echo "Java 8 expected" && exit 1
+# [[ "$javaVersion" != "1.8."* || "$javaCompilerVersion" != *"1.8."* ]] && echo "Java 8 expected" && exit 1
```

Comment:
Same here. Do you need this code for future reference?

Reply:
No. Confluence 8 doesn't support Java 8 anymore, so this check became invalid.